flang/test/Driver/omp-driver-offload.f90

```diff
@@ -33,7 +37,7 @@
 ! OFFLOAD-HOST-NOT: "-triple" "nvptx64-nvidia-cuda"
 ! OFFLOAD-HOST-NOT: "{{[^"]*}}flang-new" "-fc1" "-triple" "aarch64-unknown-linux-gnu"
 
-! RUN: not %flang -S -### %s -o %t 2>&1 \
+! RUN: %flang -S -### %s 2>&1 \
```

Comment:
Please move to a dedicate patch

Reply:
I can do, but this is still sort of the same set of changes trying to fix the test back to a state of normalcy, just slightly differently to the rest. In this case it's failing because the command expects to output multiple inputs rather than a single so the -o segment of the command breaks it and the not is then required. Although, we can leave it as an incorrect command and it'd still be able to check the correct results and yield positive! This change mostly just allows us to remove the not from the front allowing all not's to be removed from the file, making understanding of the test a little easier. 

So do please let me know if you still wish it in a seperate PR and I'll remove it and move it to a new one.

Reply:
No problem, I'll do that soon next!

Reply:
New PR with the moved test is here: https://github.com/llvm/llvm-project/pull/67335 I'll update this patch shortly to exclude the change in the other PR.